frontend/.gitignore

```diff
@@ -0,0 +1,4 @@
+# elm-package generated files
+elm-stuff
+# elm-repl generated files
+repl-temp-*
```

Comment:
This file is not generated in my environment.

```sh
$ ls -l
total 16
-rw-r--r--  1 enokawa  staff  258 Jul  4 20:50 README.md
drwxr-xr-x  3 enokawa  staff   96 Jul  4 20:53 dist
drwxr-xr-x  3 enokawa  staff   96 Jul  4 20:51 elm-stuff
-rw-r--r--  1 enokawa  staff  517 Jul  4 20:50 elm.json
drwxr-xr-x  3 enokawa  staff   96 Jul  4 20:50 src
```

Reply:
And `dist/` is not included. Don't you want it?  
I think `dist/` direcotory should not be included repo.

Reply:
Last one. Don't need a new line at the end?  
Please enable **Insert Final Newline** settings for your editor.

Reply:
please try again.
`elm make ./src/Main.elm --output=dist/index.html`